packages/devtools_app/lib/src/screens/memory/panes/chart/controller/charts/android_chart_controller.dart

```diff
@@ -0,0 +1,324 @@
+// Copyright 2024 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:devtools_shared/devtools_shared.dart';
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+
+import '../../../../../../shared/charts/chart_controller.dart';
+import '../../../../../../shared/charts/chart_trace.dart' as chart_trace;
+import '../../../../../../shared/charts/chart_trace.dart'
+    show ChartType, ChartSymbol;
+import '../../../../shared/primitives/memory_timeline.dart';
+import '../../data/charts.dart';
+
+// ignore: avoid_classes_with_only_static_members, enum-like classes are ok
+class _Color {
```

Comment:
use advanced enum instead

Reply:
Some colors are not consts, so advanced enums are not applicable :(